articles/dashboard/guides/tenants/revoke-application-signing-keys.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Revoke Application Signing Keys
+description: Learn how to revoke your tenant's application signing key, which is used to sign tokens and SAML assertions, using the Auth0 Dashboard and Auth0 Management API.
+topics:
+  - tokens
+  - access-tokens
+  - id-tokens
+  - assertions
+  - SAML
+  - signing-keys
+  - certificates
+  - dashboard
+  - mgmt-api
+contentType:
+  - how-to
+useCase:
+  - secure-api
+  - add-login
+  - call-api
+---
+
+# Revoke Application Signing Keys
+
+This guide will show you how to revoke your tenant's application signing key using the Auth0 Dashboard and Auth0 Management API. The application signing key is used to sign tokens and SAML assertions.
+
+::: warning
+Rotating your application signing key will cause all your applications to require the signing key currently in queue for validating new tokens. All previously-validated tokens will still be valid with the previous key until the previous key is revoked.
```

Comment:
Not sure this warning makes sense in this context.

Reply:
It's because revoking tokens through the dashboard requires rotation, so the warning holds true for that scenario. It may belong inside the code picker though, so it just shows for the Dashboard section.

Reply:
After reading some further comments, seems that my tenant may not be reflecting the latest updates. Will hold off on this until we talk this afternoon.